### PR TITLE
Added more JSON formats

### PR DESCRIPTION
**Pull request by andrewzolotukhin**
## Summary
Added more JSON formats. And fixed issue with 'JSONObjectEachRow' throwing an error.
Added support for the following formats: JSONStrings, JSONCompact, JSONCompactStrings, JSONColumnsWithMetadata.

## Checklist
Delete items not relevant to your PR:
- [ ] A human-readable description of the changes was provided to include in CHANGELOG

### CODE REVIEW

File: src/data_formatter/formatter.ts [hunk 1]
```diff
@@ -8,9 +8,16 @@ const streamableJSONFormats = [
...
+const notStreamableJSONFormats = [
   'JSON',
+  'JSONStrings',
+  'JSONCompact',
+  'JSONCompactStrings',
+  'JSONColumnsWithMetadata',
   'JSONObjectEachRow',
+] as const
+const supportedJSONFormats = [
+  ...notStreamableJSONFormats,
```
Comment (slvrtrn) [inline src/data_formatter/formatter.ts]: nit: `nonStreamableJSONFormats` (non as a prefix)

File: src/data_formatter/formatter.ts [hunk 2]
```diff
@@ -8,9 +8,16 @@ const streamableJSONFormats = [
   'JSONCompactStringsEachRowWithNames',
   'JSONCompactStringsEachRowWithNamesAndTypes',
 ] as const
-const supportedJSONFormats = [
+const notStreamableJSONFormats = [
   'JSON',
+  'JSONStrings',
+  'JSONCompact',
+  'JSONCompactStrings',
+  'JSONColumnsWithMetadata',
```
Comment (slvrtrn) [inline src/data_formatter/formatter.ts:16]: Could you add these new formats to `encode_values.test.ts`?

**Review by slvrtrn — COMMENTED** [review]
Thanks for the contribution! Let's add a few tests here, and we can merge this.

**Comment by andrewzolotukhin** [comment]
Hi! I've addressed your comments. Do you have any kind of release schedule? Just wondering when these changes are going to be published in npm (I going to use it in my project).

Thank You!

**Comment by slvrtrn** [comment]
@andrewzolotukhin I will release it ASAP (maybe even today).
Please stay tuned.

Again, thanks for your contribution.

**Review by slvrtrn — APPROVED** [review]

**Comment by slvrtrn** [comment]
@andrewzolotukhin 0.0.14 is out, including this changeset.

**Pull request merged**